learn/appchains/overview.md

```diff
@@ -7,43 +7,60 @@ description: Learn the high-level definitions of how a Tanssi appchain works, it
 
 ## Introduction {: #introduction }
 
-Appchains deployed through Tanssi have many [distinct features](/learn/tanssi/overview/#what-tanssi-provides){target=\_blank}, such as block production as a service.
+Appchains deployed through Tanssi receive many [benefits](/learn/tanssi/overview/#what-tanssi-provides){target=\_blank}, like block production as a service, data retrievability as a service, and security through an [external security providers](/learn/tanssi/external-security-providers/){target=\_blank} such as [Symbiotic](https://symbiotic.fi/){target=\_blank} on Ethereum.
 
-However, because Tanssi appchains are built with a [modular framework](/learn/framework/){target=\_blank}, they offer many other unique advantages at a runtime level that make them quite extensible. This [unique modularity](/learn/framework/modules/){target=\_blank} allows a developer to add functionality directly into the runtime or extend the capabilities of the EVM itself via precompiled contracts.
+Also, because Tanssi appchains are based on a [modular tech stack](/learn/framework/){target=\_blank}, they profit from unique advantages when customizations are required at a runtime level. This [modularity](/learn/framework/modules/){target=\_blank} allows developers to add functionality directly into the runtime or extend the capabilities of the EVM itself via precompiled contracts.
 
-For example, by including modules from [Frontier](https://github.com/paritytech/frontier){target=\_blank}, which is Substrate's Ethereum compatibility layer, a Tanssi appchain can be fully Ethereum-compatible, like [Moonbeam](https://moonbeam.network){target=\_blank}.
+For example, by including modules from [Frontier](https://github.com/paritytech/frontier){target=\_blank}, a Tanssi appchain can become fully Ethereum-compatible, just like [Moonbeam](https://moonbeam.network){target=\_blank}.
```

Comment:
I would higlight that Tanssi offers this as a template, consequently deploying an EVM appchain is extremely easy